gcm_toolkit/tests/test_interface.py

```diff
@@ -110,6 +110,10 @@ def test_prt_interface(petitradtrans_testdata, all_raw_testdata):
     with pytest.raises(NotImplementedError):
         interface.set_data(time=expected["times"][-1])
 
+    # test transit calculation
+    wave, spectra = interface.calc_transit_spectrum(mmw=2.33, lat_points=1)
+    assert sum(wave) == 28.373223101833855
```

Comment:
I think thats perfectly fine for now. If you would like to extend it at some point you could repeat the calculation here with the classical prt way to do it and check that `interface.calc_transit_spectrum(mmw=2.33, lat_points=1)` gives the same result

Reply:
It is not a perfect test but it runs through the file and would flag fatal errors. I will see if there is a better one.